internal/manifest/v1beta1/test-config-files/empty-configs.yaml

```diff
@@ -0,0 +1,5 @@
+# Samples Config
```

Comment:
it's not a recommended approach to put test-related data together with production code, instead you can put it under `pkg/test_samples`